fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -156,18 +150,18 @@ RdfStream getTriples(final IdentifierConverter<Resource, FedoraResource> idTrans
                          final Set<? extends TripleCategory> contexts);
 
     /**
-     * Get the JCR Base version for the node
+     * Get the base version for the node
      *
      * @return base version
      */
-    public Version getBaseVersion();
+    FedoraResource getBaseVersion();
 
     /**
-     * Get JCR VersionHistory for the node.
+     * Get a stream of version identifiers associated with this resource
```

Comment:
Comment should be updated to reflect the fact that `FedoraVersion`s are returned, not identifiers.